datafusion/sql/tests/cases/plan_to_sql.rs

```diff
@@ -697,7 +739,7 @@ fn test_table_scan_pushdown() -> Result<()> {
         plan_to_sql(&table_scan_with_projection_alias)?;
     assert_eq!(
         format!("{}", table_scan_with_projection_alias),
-        "SELECT ta.id, ta.age FROM t1 AS ta"
+        "SELECT * FROM (SELECT ta.id, ta.age FROM t1 AS ta) AS ta"
```

Comment:
That's not an improvement, right?
I am OK if this is necessary, but is there a different way to fix the original problem, without affecting here?

Reply:
Thanks @findepi. Sorry for replying so late 🙇 
You're right. I shouldn't break the result of the table scan with any pushdown. I stopped projecting a wildcard if the `SubqueryAlias`'s child is a pushdown table scan.